src/modules/dashboard/components/ExpenditurePage/ExpenditureSettings/LockedExpenditureSettings.css

```diff
@@ -0,0 +1,120 @@
+@value paddingTop: 75px;
+@value paddingRightLeftBottom: 54px;
+
+.container {
+  padding: paddingTop paddingRightLeftBottom paddingRightLeftBottom;
+}
+
+.container i {
+  /* The negative margin is used here to adjust the position of the caret icon */
+  margin-bottom: -4px;
+}
+
+.settingsRow {
+  composes: flexContainerRow flexAlignCenter flexJustifyBetween from '~styles/layout.css';
+  min-height: 41px;
+}
+
+.settingsRow button[aria-expanded="true"] {
+  padding: 2px 0 2px 6px;
+  border-radius: 4px;
+  background-color: rgb(229, 230, 247);
```

Comment:
Can we please replace all instances of this color with this variable `--grey-blue-1`